src/app/globals.css

```
@@ -1522,4 +1536,21 @@ body:has(.card-detail-modal) .workspace-grid-container {
   to {
     opacity: 1;
   }
+}
+
+/* User message breathe-in animation */
+@keyframes breatheIn {
```

Stylelint (17.3.0)

[error] Expected keyframe name "breatheIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)